treeherder/intermittents_commenter/commenter.py

```diff
@@ -452,15 +450,12 @@ def build_bug_map(self, bugs, option_collection_map, start_day, end_day):
             if test_name:
                 manifest = all_tests[test_name][0]
                 if manifest:
-                    """
-                    # Bug 1972307 - disabled due to no comments
-                    tasks_count = self.get_task_labels_and_count(manifest, start_day, end_day)
+                    tasks_count = self.get_task_labels_and_count(manifest, days)
                     job_name = bug["job__signature__job_type_name"]
                     for task_name, count in tasks_count.items():
                         if task_name == job_name or task_name == job_name.rsplit("-", 1)[0]:
```

Comment:
can we do the `job_name.rsplit("-", 1)[0]` once before the for loop?  I believe in tasks_count (coming from test-run-info.json) the task names do not have the -X, so we could simplify more:
if job_name ends with -<int>:
  job_name = job_name strip -<int>

then:
for task_name, count in tasks_count.items():
  if task_name == job_name